qemu/tests/cfg/device_assignment.cfg

```diff
@@ -7,7 +7,19 @@
     variants:
         - multi_pf_boot:
             type = boot
-            nics = "nic1 nic2"
+            nics = "nic1 nic2 nic3"
+            pci_assignable_nic1 = no
+            nic_model_nic1 = virtio
+            driver_option = 0
+            variants:
+                - 82576:
+                    pf_filter_re = 82576
+                - XL710:
+                    pf_filter_re = XL710
+                - 82599:
+                    pf_filter_re = 82599
+                - BCM57810:
+                    pf_filter_re = BCM57810
```

Comment:
@balamuruhans  i think it is ok here. The old design list only one nic, and request tester to change "pf_filter_re" according to the test environment.  The new design list more nics, so i think it can lighten the workload.
If no nic match with the test environment, tester can change it again, this is same with the old design. what do you think ?